cloudslang-content-verifier/src/main/java/io/cloudslang/lang/tools/build/commands/ApplicationArgs.java

```diff
@@ -44,6 +44,14 @@
             description = "Whether or not test coverage data should be outputted")
     public boolean coverage = false;
 
+    @Parameter(names = {"--parallel", "-par"},
+            description = "Whether or not parallel test execution should be used")
+    public boolean parallel = false;
+
+    @Parameter(names = {"--thread-count", "-th"},
+            description = "Number of threads to be used in case of parallel test execution. Has no effect for sequential execution. By default, it is set to the number of number of processors available to the SlangBuilder.")
```

Comment:
duplicate `number of`
IMHO cut the `available to the SlangBuilder` - this may not be relevant information for the user in the console